### PR TITLE
Day/01/phase1

### PR DESCRIPTION
**Pull request by madvid**
I just have to create a last graph for the last exercise (ex03) and I finish all I wanted to do, thus you can already check what I did.

### CODE REVIEW

File: day01/ex00/ex00.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+# Exercise 00 - Linear Regression
+
+|                         |                    |
+| -----------------------:| ------------------ |
+|   Turnin directory :    |  ex00              |
+|   Files to turn in :    |  linear\_model.py  |
+|   Authorize module :    |  numpy, matplotlib |
```
Comment (matboivin) [inline day01/ex00/ex00.md]: Authorize module become Authorized modules

File: day01/ex00/ex00.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+|   Forbidden module :    |  sklearn           |
```
Comment (matboivin) [inline day01/ex00/ex00.md]: Forbidden module becomes Forbidden modules

**Comment by madvid** [comment]
Nice, thank you, I'll do it right after the merge

File: day01/ex00/ex00.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+
+As hypothesis function h, you will choose:
+$$
+h(X)= \theta_0x_0 + \theta_1x_1
+$$
+where $\pmb{X}$ is the vector $(x_0, x_1)$ and $\pmb{theta} = (theta_0, \theta_1)$
+
+
+You will model the data and plot 2 differents graphs:
+* The graph with the data and the best hypothesis you find for the spacecraft piloting score versus the quantity of "blue pills" (see example figure 1),
```
Comment (ppeigne) [inline day01/ex00/ex00.md]: May be we should be more explicit about the fact they have to fit it "by hand" ?

File: day01/ex01/ex01.md [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+|   Turnin directory :    |  ex01              |
+|   Files to turn in :    |  multi_linear\_model.py  |
+|   Authorize module :    |  numpy, matplotlib |
+|   Forbidden module :    |  sklearn           |
+|   Forbidden function :  |  LinearRegression  |
+|   Remarks :             |  Read the doc      |
+
+**Objectives:** 
+
+* Reinforce the mathematical skills tackled in **Mathematical Delights**, especially the vectorized form of __Root Mean Square__ and __gradient descent__ ($\text{\smallIf you were to lazy to do it in the exercise 00 \textit{cof cof}}$).
```
Comment (ppeigne) [inline day01/ex01/ex01.md]: They do not have to code gradient_descent in 00 but only .predict()

File: day01/ex02/ex02.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
+# Exercise 02 - Mutiples features and Normal Equation
```
Comment (ppeigne) [inline day01/ex02/ex02.md]: The mutliple features are done in the previous exo. The name is may be not appropriate anymore ?

**Pull request merged**